content/md/supplier-onboarder/manage-products/update-products.md

```diff
@@ -28,6 +28,7 @@ The header panel shows the following product information:
 * A breadcrumb to navigate within the [product model and variants](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html)
 * The number of missing attributes
 * Some filters to directly access attribute groups or missing attributes
+* Comment on the product if there is
```

Comment:
It looks like the sentence is not finished here?

Reply:
Same reaction

Reply:
"Comment on the product if there is one" would be better

Reply:
changed, tell me if it's OK